foundry/entity.d.ts

```diff
@@ -440,7 +447,7 @@ declare class Entity<D extends Entity.Data = Entity.Data> {
    * @param id           - The numeric ID of the child to retrieve
    * @param strict       - Throw an Error if the requested id does not exist, otherwise return null. Default false.
    */
-  getEmbeddedEntity(embeddedName: string, id: string, { strict }?: { strict?: boolean }): any;
+  getEmbeddedEntity(embeddedName: string, id: string, { strict }?: { strict?: boolean }): EmbeddedEntity.Data;
```

Comment:
That's incorrect, not all embedded entities actually extend `EmbeddedEntity` (actually only `ActiveEffect` does....).